src/web/components/Onwards/OnwardsTitle.tsx

```diff
@@ -5,39 +5,67 @@ import { palette } from '@guardian/src-foundations';
 import { headline } from '@guardian/src-foundations/typography';
 import { from } from '@guardian/src-foundations/mq';
 
+const linkStyles = css`
+    text-decoration: none;
+    color: ${palette.neutral[7]};
+
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+const TitleWrapper = ({
+    url,
+    children,
+}: {
+    url?: string;
+    children: JSXElements;
+}) =>
+    url ? (
+        <a className={linkStyles} href={url}>
+            {children}
+        </a>
+    ) : (
+        <>{children}</>
+    );
+
 export const OnwardsTitle = ({
     title,
     description,
+    url,
 }: {
     title: string;
     description?: string;
+    url?: string;
 }) => (
     <>
-        <h2
-            className={css`
-                ${headline.xsmall()};
-                color: ${palette.neutral[7]};
-                font-weight: 900;
-                padding-bottom: 14px;
-                padding-top: 6px;
-
-                ${from.leftCol} {
+        <TitleWrapper url={url}>
+            <h2
+                className={css`
```

Comment:
Rather than the abstraction of `TitleWrapper` it would be easier to read if we just inline the logic, a bit like:

```
                {url ? (
                    <a>
                        <h2 className={headerStyles}>{title}</h2>
                    </a>
                ) : (
                    <h2 className={headerStyles}>{title}</h2>
                )}
```

Sometimes repeating code is a good thing. Although we now have two h2 tags the same it's actually less code and more readable plus if we need to change anything later on future developers won't start doing the passing a prop thing to handle changes for one use case and not the other

An example of this elsewhere in the codebase is: https://github.com/guardian/dotcom-rendering/blob/7c709ce4a9ecdb766c05b1dd050b5ea41dfadf5b/src/web/components/LinkHeadline.tsx#L78

Sorry, I went on a bit there. You get the idea!